src/neptune/metadata_containers/project.py

```diff
@@ -406,7 +420,9 @@ def fetch_models_table(self, *, columns: Optional[Iterable[str]] = None) -> Tabl
                 name="sys/trashed",
                 type=NQLAttributeType.BOOLEAN,
                 operator=NQLAttributeOperator.EQUALS,
-                value=False,
-            ),
+                value=trashed,
+            )
+            if trashed is not None
+            else None,
```

Comment:
I'm not sure here (needs to be checked) but probably `NQLEmptyQuery` seems to be more expected than `None`.